src/main/kotlin/com/openlattice/search/renderers/CAREIssueAlertEmailRenderer.kt

```diff
@@ -0,0 +1,186 @@
+package com.openlattice.search.renderers
+
+import com.google.common.collect.Maps
+import com.openlattice.data.requests.NeighborEntityDetails
+import com.openlattice.mail.RenderableEmailRequest
+import com.openlattice.search.requests.PersistentSearch
+import org.apache.olingo.commons.api.edm.FullQualifiedName
+import java.time.OffsetDateTime
+import java.util.*
+
+private const val FROM_EMAIL = "courier@openlattice.com"
+private const val TEMPLATE_PATH = "mail/templates/shared/CAREIssueAlertTemplate.mustache"
+
+/* FQNs */
+private val PERSON_ID_FQN = FullQualifiedName("nc.SubjectIdentification")
+private val FIRST_NAME_FQN = FullQualifiedName("nc.PersonGivenName")
+private val MIDDLE_NAME_FQN = FullQualifiedName("nc.PersonMiddleName")
+private val LAST_NAME_FQN = FullQualifiedName("nc.PersonSurName")
+private val STATUS_FQN = FullQualifiedName("ol.status")
+private val DESCRIPTION_FQN = FullQualifiedName("ol.description")
+private val PRIORITY_FQN = FullQualifiedName("ol.priority")
+private val CATEGORY_FQN = FullQualifiedName("ol.category")
+private val TITLE_FQN = FullQualifiedName("ol.title")
+private val COMPLETED_DATE_TIME_FQN = FullQualifiedName("date.completeddatetime")
+private val ENTRY_UPDATED_FQN = FullQualifiedName("general.entryupdated")
+private val OPENLATTICE_ID_FQN = FullQualifiedName("openlattice.@id")
```

Comment:
There's got to be a more official way of doing this

Reply:
I think this should work:

```
import com.openlattice.edm.EdmConstants.Companion.ID_FQN
```

Reply:
`EdmConstants.ID_FQN`